hatchery/src/world.rs

```diff
@@ -76,13 +77,47 @@ impl World {
         )))))
     }
 
+    pub fn persist(&self) -> Result<(), Error> {
```

Comment:
This should probably also return a SnapshotId?

Reply:
sounds good

Reply:
issue is that there are multiple snapshots, one per each module, we don't have global snapshot (yet), so I can't return snapshot id in this case